app/concerns/sfx_handler.rb

```diff
@@ -54,15 +52,17 @@ def url_constructor(scan: false)
     # for request items, but this ends up with scan requests for Technique
     # (the yearbook) being routed to a Polish land use journal. Call number
     # fixes this bug and does not seem to introduce new ones.
-    encoded_call_no = ERB::Util.url_encode(@call_number)
+    encoded_call_no = URI.encode_www_form_component(@call_number)
 
     url_parts = [
       sfx_host.to_s,
       "?sid=ALEPH:#{analytics}",
       "&amp;call_number=#{encoded_call_no}",
       "&amp;barcode=#{@barcode}",
-      "&amp;title=#{encoded_title}",
-      "&amp;location=#{encoded_location}"
+      "&amp;title=#{URI.encode_www_form_component(@title)}",
```

Comment:
I don't particularly care if we do the encoding here during the interpolation or earlier, but it seems like call_no and title should be handled the same way.

Reply:
I didn't move the call number in because of the documentation comment about it whereas the title had no comments so it felt okay to just move it inline.